### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by qitianyuu**
change code

### CODE REVIEW

File: mcr.cpp [hunk 1]
```diff
@@ -1,53 +1,104 @@
...
+    system("clear");
+    std::cout << "X = Player 1" << std::endl << "O = Player 2" << std::endl;
+    std::cout << " " << game[0][0] << " | " << game[0][1] << " | " << game[0][2] << std::endl;
+    std::cout << " -- --- -- " << std::endl;
+    std::cout << " " << game[1][0] << " | " << game[1][1] << " | " << game[1][2] << std::endl;
+    std::cout << " -- --- -- " << std::endl;
+    std::cout << " " << game[2][0] << " | " << game[2][1] << " | " << game[2][2] << std::endl;
+}
+
+bool is_right_input(int i, int j){
```
Comment (chenjunyi1999) [inline mcr.cpp:32]: good!

File: mcr.cpp [hunk 2]
```diff
@@ -1,53 +1,104 @@
 #include <iostream>
 #include <stdio.h>
+#include <stdlib.h>
```
Comment (canuang) [inline mcr.cpp:3]: \^o^/

**Review by chenjunyi1999 — COMMENTED** [review]
it is good

**Review by canuang — CHANGES_REQUESTED** [review]

**Review by canuang — COMMENTED** [review]
comment

**Pull request merged**